site/content/docs/user/loadbalancer.md

```diff
@@ -6,68 +6,29 @@ menu:
     identifier: "user-loadbalancer"
     weight: 3
 description: |-
-    This guide covers how to get service of type LoadBalancer working in a kind cluster using [Metallb].
+    This guide covers how to get service of type LoadBalancer working in a kind cluster using [Cloud Provider KIND].
 
-    This guide complements MetalLB [installation docs], and sets up MetalLB using layer2 protocol.  For other protocols check MetalLB [configuration docs].
+    This guide complements Cloud Provider KIND [installation docs].
 
-    With Docker on Linux, you can send traffic directly to the loadbalancer's external IP if the IP space is within the docker IP space.  
-    
-    On macOS and Windows, docker does not expose the docker network to the host.  Because of this limitation, containers (including kind nodes) are only reachable from the host via port-forwards, however other containers/pods can reach other things running in docker including loadbalancers.  You may want to check out the [Ingress Guide] as a cross-platform workaround.  You can also expose pods and services using extra port mappings as shown in the extra port mappings section of the [Configuration Guide].
-    
-
-    [MetalLB]: https://metallb.universe.tf/
-    [installation docs]: https://metallb.universe.tf/installation/
-    [configuration docs]: https://metallb.universe.tf/configuration/
+    [Cloud Provider KIND]: https://github.com/kubernetes-sigs/cloud-provider-kind
+    [installation docs]: https://github.com/kubernetes-sigs/cloud-provider-kind?tab=readme-ov-file#install
 
     [Ingress Guide]: /docs/user/ingress
     [Configuration Guide]: /docs/user/configuration#extra-port-mappings
 
 ---
 
-## Installing MetalLB using default manifests
-
-### Apply MetalLB manifest
-
-Since version 0.13.0, MetalLB is configured via CRs and the original way of configuring it via a ConfigMap based configuration
-is not working anymore.
-
-{{< codeFromInline lang="bash" >}}
-kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.13.7/config/manifests/metallb-native.yaml
-{{< /codeFromInline >}}
+## Installing Cloud Provider KIND
 
-Wait until the MetalLB pods (controller and speakers) are ready:
+Cloud Provider KIND can be installed using golang
 
 {{< codeFromInline lang="bash" >}}
-kubectl wait --namespace metallb-system \
-                --for=condition=ready pod \
-                --selector=app=metallb \
-                --timeout=90s
+go install sigs.k8s.io/cloud-provider-kind@latest
```

Comment:
Are we publishing anywhere that we can use kubectl instead of `go install`? I suppose the target audience for `kind` will have `go` available, but it would be nice to not have to have the whole go toolchain installed to use Cloud Provider Kind.

The use of `latest` gives me a little concern too, just because we try to discourage the use of latest elsewhere when installing things and prefer to be more explicit.

Reply:
latest in `go install` means the latest released version https://pkg.go.dev/cmd/go#hdr-Compile_and_install_packages_and_dependencies , is not like in containers that is unspecified

Reply:
yeah this is confusing versus docker, go consistently means "highest release tag" when using the latest placeholder, which should be fine here. We didn't pin to a particular metallb version in the docs.

users should absolutely consider pinning a specific version but i'm not sure we want to do that here, we want to maintain compatibility with kind and that should be fairly easy since it only minimally interacts (famous last words) and mostly talks to docker/podman and the api server

Reply:
We are testing kubernetes HEAD with cloud-provider-kind HEAD so this combination MUST work or we'll have a bigger problem:)